docs/guides-agent-app.md

```diff
@@ -310,19 +293,43 @@ You can also verify that permissions have been set properly through the UI:
    <img width="800" src="/docs/assets/agent-guide/agent-6.png">
 </p>
 
-3.  Under **Permissions set on this app** you should see that Voting app is also allowed to execute actions on behalf of the Agent app.
+3.  Under **Permissions set on this app** you should see that Voting app now has permission to use the Agent app to execute actions. 🎉🎉🎉
 
 <p align="center">
    <img width="800" src="/docs/assets/agent-guide/agent-7.png">
 </p>
 
-# Use cases
+# Interacting with Aragon Agent
+
+## Introducing `dao act`
+
+`dao act` is the main way we'll interact with Aragon Agent. It's the command we use to perform transactions with the Agent app directly from aragonCLI.
+
+According to the documentation:
+
+>`dao act` provides some syntax sugar over `dao exec` for executing actions using Agent app instances in a DAO.
+
+Like `dao exec` it takes at least three arguments:
+
+1. The address of the Agent app you want to use to perform an action.
 
-## A. Voting in another organization
+2. The address of an **external contract** or **the address of an app** within a DAO.
+
+3. The [full signature](https://developer.mozilla.org/en-US/docs/Glossary/Signature/Function) of the method we wish to execute in either the external contract or the app we specified in the second arugument.
```

Comment:
This links to something unrelated to Ethereum function signatures, this is the best resource I could find about them: https://www.4byte.directory

Reply:
Not sure it's immediately clear from that resource that we mean the human readable function signature. This might be more useful: https://solidity.readthedocs.io/en/v0.5.3/abi-spec.html#function-selector

What do you think?

Reply:
Yes, I think is best to link both sources 👍